lib/Parser/Parse.cpp

```diff
@@ -2436,19 +2436,26 @@ bool Parser::IsImportOrExportStatementValidHere()
 }
 
 template<bool buildAST>
-ParseNodePtr Parser::ParseImportDeclaration()
+ParseNodePtr Parser::ParseImport()
 {
     Assert(m_scriptContext->GetConfig()->IsES6ModuleEnabled());
     Assert(m_token.tk == tkIMPORT);
 
+    m_pscan->Scan();
+
+    // import()
+    if (m_token.tk == tkLParen)
+    {
+        ParseNodePtr specifier = ParseTerm<buildAST>();
+        return CreateCallNode(knopCall, CreateNodeWithScanner<knopImport>(), specifier);
```

Comment:
Added dynamic-module-import-specifier.js
More to come.

---
In reply to: [114915006](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114915006) [](ancestors = 114915006)